test/test_helper.rb

```diff
@@ -47,9 +47,8 @@ def serialization_options(options)
 
 require 'support/rails5_shims'
 
-require 'fixtures/active_record'
-
-require 'fixtures/poro'
+require 'support/fixtures/active_record'
+require 'support/fixtures/poro'
```

Comment:
I think the Rails convention is for fixtures to go in the test directory http://guides.rubyonrails.org/testing.html#the-low-down-on-fixtures though admittedly these are not the same datatype as referenced in the link and I have seen both file locations...  thoughts?